pxr/imaging/hdsi/coordSysPrimSceneIndex.cpp

```diff
@@ -302,7 +302,7 @@ HdsiCoordSysPrimSceneIndex::GetChildPrimPaths(const SdfPath &primPath) const
     if (it == _targetedPrimToNameToRefCount.end()) {
         return result;
     }
-    for (const std::pair<TfToken, size_t> &nameAndRefCount : it->second) {
+    for (const std::pair<TfToken, size_t> nameAndRefCount : it->second) {
```

Comment:
You should be able to avoid the copy by making `nameAndRefCount` `const auto&` or `const std::pair<const TfToken, size_t>&.

Reply:
Both worked, will update.